### PR TITLE
Updated about and personal pages.

### PR DESCRIPTION
**Pull request by Steven-Bonnell-II**

### CODE REVIEW

**Comment by UnderAndOver** [comment]
image

**Comment by cantclosevim** [comment]
looks good, merge it :^)

**Comment by vlanx** [comment]
LGTM!

File: docs/personal.md [hunk 1]
```diff
@@ -5,16 +5,27 @@ title: Personal
...
 
-I have a son called Nathaniel with my ex-girlfriend, Rachel. My ex-girlfriend and I broke up a long time ago because we had an incredibly toxic relationship, but we are on very good terms today and get along well with each other. I have an ex-wife as well, but she is not my ex-girlfriend and she is not the mother of my child.
+I have a son called Nathaniel with my ex-girlfriend, Rachel. My ex-girlfriend and I broke up a long time ago due to our toxic relationship, but we are on good terms today and get along well with each other. I have an ex-wife as well, but she is not the mother of my child.
 
 ## My Relationship
 
 [Melina](https://www.twitch.tv/melina) is my wife. We met in New Zealand when she was 20 years old and I was 30 years old.
 
-Melina and I are currently in an open/poly relationship. We treat each other as primary partners, though we may pursue other sexual/romantic relationships as well.
+Melina and I are currently in an open/poly relationship. We treat each other as primary partners, though we may pursue other sexual/romantic relationships as well.8
```
Comment (frkanm) [inline docs/personal.md:20]: good one bonnell "well.8"

**Comment by JesseNaser** [comment]
PEPE WON BIG

File: docs/personal.md [hunk 2]
```diff
@@ -5,16 +5,27 @@ title: Personal
...
+
+## Links
+
+* [YouTube](https://www.youtube.com/user/destiny/)
+* [Instagram](https://www.instagram.com/destiny/)
+* [Twitter](https://www.twitter.com/TheOmniLiberal)
+* [Rumble](https://www.rumble.com/Destiny)
+* [Kick](https://www.kick.com/destiny)
+* [Reddit](https://www.reddit.com/r/Destiny/)
+* [Email](contact@destiny.gg)
```
Comment (explodingcamera) [inline docs/personal.md]:
```suggestion
* [Email](mailto:contact@destiny.gg)
```

**Comment by Voiture-0** [comment]
Looks good to me! go ahead and merge it

File: docs/personal.md (hunk 1 above)
Comment (joshrzemien) [reply docs/personal.md:20]: poor quality merge request please learn to code and try again

**Comment by Anonymeowse** [comment]
Approved. Ship it. :shipit:

**Comment by krshaw** [comment]
🚢 LGTM SHIPIT

**Review by krshaw — APPROVED** [review]

**Comment by nJavo** [comment]
Don't merge. Seems broken. Can you fix by EOD?

**Pull request merged**